app/main.py

```diff
@@ -1,2 +1,63 @@
+from dataclasses import dataclass
+from typing import Hashable, Any
+
+
+@dataclass
+class Node:
+    key: Hashable
+    value: any
+    key_hash: int
+
+
 class Dictionary:
-    pass
+    INITIAL_CAPACITY = 8
+    THRESHOLD = 2 / 3
```

Comment:
The `THRESHOLD` should be calculated as `2 / 3` of the capacity, but currently, it is set as a constant value `2 / 3`. It should be `self.capacity * (2 / 3)` or similar to dynamically calculate the threshold based on the current capacity.